internal-api/src/main/kotlin/com/ajaxproject/api/internal/warehousesvc/KafkaTopic.kt

```diff
@@ -0,0 +1,13 @@
+package com.ajaxproject.api.internal.warehousesvc
+
+import com.ajaxproject.api.internal.warehousesvc.NatsSubject.WAREHOUSE_PREFIX
+
+object KafkaTopic {
+
+    object Waybill {
+
+        private const val WAYBILL_PREFIX = "${WAREHOUSE_PREFIX}.waybill"
+
+        const val CREATION = "$WAYBILL_PREFIX.creation"
```

Comment:
```suggestion
        const val CREATED = "$WAYBILL_PREFIX.created"
```

Kafka event - some event has happened. So, use verb 3rd form

Reply:
Fixed